standard_readme.md

```diff
@@ -0,0 +1,97 @@
+# Name of the project
+
+- [Name of the project](#name-of-the-project)
+- [Development](#development)
+  * [Requirements](#requirements)
+  * [Set up the app the first time](#set-up-the-app-the-first-time)
+  * [Run the app](#run-the-app)
+  * [Run the tests](#run-the-tests)
+  * [Useful commands](#useful-commands)
+  * [Download production or staging DB](#download-production-or-staging-db)
+  * [Workflow](#workflow)
+  * [Documentation](#documentation)
```

Comment:
This looks like a Development documentation section, where I might put links to Rspec or Rubocop documentation.

There needs to be a place for links to things like API documentation for *this* project.
Maybe that belongs under Production? or in it's own top level section?

Reply:
The idea was to put here things like the API documentation for _this project_. You might be right that Development might not be the right category. Perhaps something we should put at the top?

I would place `Rspec` or `Rubocop` documentations under `#notable-inclusions-and-notable-exclusions`